packages/search/src/search.tsx

```diff
@@ -163,6 +164,9 @@ const InnerSearch = (
 			blur() {
 				searchInput.current?.blur();
 			},
+			updateKeyword( value: string ) {
```

Comment:
I'd personally call it `setKeyword`, because it's a classic setter 🙂

Reply:
✅